x/gov/types/params.go

```diff
@@ -26,6 +26,7 @@ var (
 	DefaultQuorum                    = sdk.NewDecWithPrec(334, 3)
 	DefaultThreshold                 = sdk.NewDecWithPrec(5, 1)
 	DefaultExpeditedThreshold        = sdk.NewDecWithPrec(667, 3)
+	DefaultExpeditedQuorumThreshold  = sdk.NewDecWithPrec(667, 3)
```

Comment:
nit: similar reason to my other comment
```suggestion
	DefaultExpeditedQuorum  = sdk.NewDecWithPrec(667, 3)
```